lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -680,35 +671,14 @@ data ErrUnXPrvStripPub
 -- - the key is an old byron key
 unXPrvStripPubCheckRoundtrip :: XPrv -> Either ErrUnXPrvStripPub ByteString
 unXPrvStripPubCheckRoundtrip k = do
-    let res = unXPrvStripPub k
-    case (fmap unXPrv . xPrvFromStrippedPubXPrv $ res) of
+    let res = xprvToBytes k
+    case (fmap CC.unXPrv . xPrvFromStrippedPubXPrv $ res) of
```

Comment:
This is just `xprvFromBytes`

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1640/commits/e9d860edc854e575712c720405e06aa515ea77f7